ONSdigital/eq-author
# Add previous answer to max value validation

### What is the context of this PR?

Implements functionality which allows users to validate max values against a previous answer for number and currency answers

### How to review 
- Run tests
- Ensure feature works as per https://trello.com/c/omc5PScG/643-validation-max-value-against-previous-answer-9

### Dependencies

- [x] https://github.com/ONSdigital/eq-author-api/pull/152
